sagemaker-pipelines/data-ingestion/feature-store-data-ingestion-via-sagemaker-model-building-pipeline.ipynb

```diff
@@ -0,0 +1,730 @@
+{
```

Comment:
<p>Maybe clarify why a "DataWrangler" processing job is used here. It might be confusing to users who don't have context here...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2585/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>